_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,194 @@
+
+__installation improvements:__ following a continued commitment to improve the useability of the CARLA simulator, the CARLA agents wheel is now __available through PyPi__ so can now be installed with pip. A new wheel for the core CARLA API library is now also available for Python version 3.9.
+
+
+## Instance Semantic Segmentation
+
+CARLA now has a brand new instance semantic segmentation sensor that outputs images containing a different ID for every object of the same type. Instance semantic IDs are now available embedded in the G and B channels of the RGB output of the sensor data, alongside the standard semantic IDs in the R channel. The new functionality provides access to the ground truth identity of objects traversing the sensor field of view, enabling training, improvement and evaluation of instance semantic segmentation networks. This new functionality brings a whole new world of possibilities for training machine learning algorithms to differentiate overlapping objects in scenes.
+
+![instance_semantic_segmentation](/img/posts/2021-11-07/instance_semantic_segmentation.png){:class="img-fluid rounded mx-auto d-block"}
+
+## Runtime texture streaming
+
+The API now exposes functionality to update textures during runtime to allow texture modification without relying on the UE4 editor. Textures can now be updated and modified programmatically through the API, allowing texture updating to occur for example during the execution of a neural network training script. This can be useful for example when training neural networks to avoid overfitting pitfalls, tuning networks to focus more on shape and form rather than color or texture. 
+
+```py
+texture = carla.TextureColor(width,height)
+for x in range(0,len(image[0])):
+    for y in range(0,len(image)):
+        color = image[y][x]
+        r = int(color[0])
+        g = int(color[1])
+        b = int(color[2])
+        a = int(color[3])
+        texture.set(x, height - y - 1, carla.Color(r,g,b,a))
+world.apply_color_texture_to_object('SM_Cartel_Add_5', carla.MaterialParameter.Diffuse, texture, 0)
+```
+
+![runtime_texture_updating](/img/posts/2021-11-07/runtime_texture.gif){:class="img-fluid rounded mx-auto d-block"}
+
+## Pedestrian ground truth skeletons
+
+The CARLA API now provides functionality to retrieve the ground truth bone positions of pedestrians in the simulation. This opens up a wealth of possibilities for training and evaluation of human pose estimation models for use in autonomous driving. A CARLA vehicle can now record the bone positions of nearby pedestrians to compliment it's sensor data with access to the ground truth pose for comparison with model-estimated poses. This new functionality also allows easy augmentation of pedestrian behavior with custom guestures blended into the default animations like waving or pointing. 
```

Comment:
> provides functionality

provides THE functionality

> pedestrians to compliment it's sensor

COMPLEMENT

> guestures

GESTURES

Reply:
complement and gestures changed not sure about the "THE"